daemon/process/settings/dobby.xi6.json

```diff
@@ -31,7 +31,8 @@
   },
 
   "network": {
-    "externalInterfaces": [ "eth0", "wlan0" ]
+    "externalInterfaces": [ "eth0", "wlan0" ],
```

Comment:
If the network interface doesn't exist, then Network setup will fail in step 8 of `setupBridgeDevice` where we try to change settings of the external interfaces, causing the plugin to stop processing and exit with an error.

If the device is set to a 'DOWN' state or otherwise non-functional, it will run happily and not complain. I personally don't think that's something Dobby should be worrying about.

Reply:
That makes sense - agree if the interface is DOWN then that's not Dobby's job to sort out as they're are probably bigger issues